core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -83,12 +83,13 @@ object ConsumerGroupCommand extends Logging {
                 printAssignment(assignments, !opts.useOldConsumer)
               case Some("Stable") =>
                 printAssignment(assignments, !opts.useOldConsumer)
+              case None =>
```

Comment:
Minor: any reason to change the ordering here. It's better to have the case None as the last block in here.